tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist=py27, lint
+
+[testenv]
+setenv =
+    PYTHONHASHSEED=0
+    DATADOG_API_KEY=mock_api_key
+    DATA_KENNEL_APP_KEY=mock_app_key
+deps=-rtest-requirements.pip
+commands=nosetests
+
+[testenv:lint]
+basepython=python2.7
+deps=-rtest-requirements.pip
+commands=
+    pylint --rcfile=pylintrc --output-format=colorized data_kennel test
+    pycodestyle --config pep8rc
```

Comment:
Also weren't you going to move the pep8rc stuff to this file?

Reply:
i think it belongs in `setup.cfg` instead of here